exercises/knapsack/description.md

```diff
@@ -0,0 +1,10 @@
+Given a set of items, each with a weight, w and a value, v, determine the number of each item to include in a collection so that the total weight is less than or equal to a given limit and the total value is as large as possible
```

Comment:
this sentence says "determine the number of each item", but that is not consistent with the "Answer is 90" line below. If the answer is 90, we determined the maximum value, not the number of items. The two sentences should be made consistent with each other (and also with canonical-data.json).

Reply:
Yup changed as well!